### PR TITLE
[Testing:TAGrading] access control for submitted pdfs

### PR DESCRIPTION
**Pull request by jchulton**
### Please check if the PR fulfills these requirements:

* [ ] The PR title and message follows our [guidelines](https://submitty.org/developer/how_to_contribute)
* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant

### What is the current behavior?
There is only simple testing for pdfs, which only covers instructors

### What is the new behavior?
More extensive e2e testing test peer, and non-instructor pdf grading

### CODE REVIEW

**Comment by jchulton** [comment]
This branch introduces and relies on sample courses with peer gradeables. In order to test this branch, you will have to run recreate_sample_courses.py

File: tests/e2e/test_pdfs.py [hunk 1]
```diff
@@ -1,22 +1,71 @@
...
-        self.driver.find_element_by_xpath('//a[contains(@file-url,"words_249.pdf")]').click()
-        self.driver.implicitly_wait(10)
+        current_window = self.driver.window_handles[0]
+        self.driver.find_element_by_id('open_file_'+pdf_name).click()
+        new_window = self.driver.window_handles[1]
+        self.driver.switch_to.window(new_window)
+        #self.wait_for_element((By.ID, "content"))
+        #text = self.driver.find_element_by_id("content")
+        #self.assertFalse("You don't have access to this page." in text)
+        self.driver.close()
```
[the remark this answers is not in the document]
Comment (jchulton) [reply tests/e2e/test_pdfs.py]: Removed commented code

File: tests/e2e/test_pdfs.py [hunk 2]
```diff
@@ -1,22 +1,71 @@
...
+        self.driver.implicitly_wait(20)
         self.driver.find_element_by_id('pageContainer1')
+    def switch_settings(self, setting):
+        self.log_out()
+        self.log_in(user_id="instructor")
+        self.click_class('sample')
+        self.driver.find_element_by_xpath('//a[contains(@href,"/sample/gradeable/grading_homework_pdf/update")]').click()
+        self.driver.find_element_by_id('page_3_nav').click()
+        select_element = Select(self.driver.find_element_by_id("minimum_grading_group"))
+        select_element.select_by_visible_text(setting);
```
[the remark this answers is not in the document]
Comment (jchulton) [reply tests/e2e/test_pdfs.py]: Removed semi-colons

**Comment by jchulton** [comment]
The tests are erroring because you can't view the files on the TA Grading Interface, so they should error. The displaying of anon_ids seems unrelated to this PR to me.

**Comment by jchulton** [comment]
I've fixed the failing tests and pushed some quick fixes. The displaying anon_ids is a discussion for another branch, since this branch doesn't deal with how anon_ids are displayed. The developer console error is on this branch and master and seems unrelated. The new TA Grading interface bug was fixed my merging master

**Comment by Kevin-MM** [comment]
Hi @jchulton can you correct the tests errors?

**Comment by bmcutler** [comment]
@jchulton please resolve conflicts

**Comment by bmcutler** [comment]
@jchulton please resolve conflicts

**Pull request merged**